ug/instructors.md

```diff
@@ -56,3 +77,14 @@ The repositories used for initial bug reporting will be created automatically by
   This must be a public repo with write access given to students.
 * A repo to hold bugs for tutors to moderate e.g., `module-org/pe-moderation`<br>
   This can be a private repo with write access given to tutors.
+  
+
+## Creating a session link for students
+
+A list of profiles can be modified in the Profile repository. 
+
+To create a session link, instructors will need to encode this url `https://catcher-org.github.io/CATcher/?session=PROFILE_NAME` where `PROFILE_NAME` is the profile name of the session. This profile name must correspond exactly to the profile name in the Profile repository. 
+
+For example, for profile name `"CS2103/T Alpha Test"`, the url will be `https://catcher-org.github.io/CATcher/?session=CS2103%2FT%20Alpha%20Test`.
+
+If the profile name is valid, students can access the link to select the corresponding session automatically. Students can then click the Submit button to enter the intended session without selecting from the dropdown.
```

Comment:
Let's add a modifier on **Submit** to highlight its semantics, maybe bold

I think md files also need a trailing newline as convention? can double check that

Reply:
Thanks, 

Also seems like it's not a md file convention but a general convention for text files https://unix.stackexchange.com/a/18789